src/renderers/render.js

```diff
@@ -1,24 +1,36 @@
 import { buildComponentFromVNode } from '../isomorphic/createComponent';
-import RenderUtils from '../utils/render-utils';
 
-let FiberComponent;
+let component;
+let initialVnode;
 
+/**
+ * @public
+ * @param vnode
+ * @param parent
+ */
 const render = (vnode, parent) => {
-	if (vnode == null || typeof vnode === 'boolean') vnode = '';
+	initialVnode = vnode;
```

Comment:
vnode and initialVnode are always equal inside irender?

Reply:
Yes, the use case of `initialVnode` is only when `buildComponentFromVNode` returns `null` or `undefined`, but this has been removed in #44 and may be more clear to you.